dynamic_programming/golmine.py

```diff
@@ -0,0 +1,32 @@
+# Goldmine Problem.
+
+#Question Problem : https://leetcode.com/problems/path-with-maximum-gold/
+
+n = int(input().strip())
+m = int(input().strip())
+arr = [[0]*m for _ in range(n)]
+print(arr)
+for i in range(n):
+    arr[i] = [int(j) for j in input().strip().split(" ")]
+
+dp = [[0]*m for _ in range(n)]
```

Comment:
Do not use abbreviations for variable names. Readability is key when writing code